blog/page1.md

```diff
@@ -1,54 +1,62 @@
-# Creating a File Sharing x Conversion Web App with Azure Applications + Online Convert API + Repl.it
+# Creating a File Sharing and Conversion Web App with Azure Applications, Online Convert API, and Repl.it
 
-> If you were a student, knew a student, or had a student that participated in this year's AP Collegeboard Exams, you may have either witnessed or felt the stress of submitting hand written work *virtually* and within a small time constraint.
-![stress](https://media.giphy.com/media/3o7TKRwpns23QMNNiE/giphy.gif)
+If you or someone you know participated in this year's AP Collegeboard Exams, you probably recognize the stress of submitting hand written work *virtually* within a small time constraint.
+![alt text here](https://media.giphy.com/media/3o7TKRwpns23QMNNiE/giphy.gif)
```

Comment:
All of your images should have alt-text on them, please see: https://support.microsoft.com/en-us/office/everything-you-need-to-know-to-write-effective-alt-text-df98f884-ca3d-456c-807b-1a1fa82f5dc2